### PR TITLE
test loops with invalid markup

### PR DESCRIPTION
**Pull request by Autre31415**
This will also print some warnings in the console to increase code coverage

### CODE REVIEW

File: test/looping.js [hunk 1]
```diff
@@ -128,14 +128,14 @@ describe('Looping', function() {
   });
 
   it('should ignore loop with invalid through attribute (looping/undefinedObjectLoop.html)', function(done) {
+    teddy.setVerbosity(2);
     assert.equalIgnoreSpaces(teddy.render('looping/undefinedObjectLoop.html', model), '<div></div>');
+    teddy.setVerbosity(1);
```
Comment (kethinov) [inline test/looping.js]: oh to get it to trigger the warning code path

Comment (kethinov) [inline test/looping.js]: can we make mocha suppress the display of the console output while still triggering the warn code path?

Comment (Autre31415) [inline test/looping.js]: This is related to bringing back the invalid through var warning in #120.

I'm setting verbosity to 2 so the warning prints and that code line is covered

Comment (Autre31415) [inline test/looping.js]: lol beat me to it there

File: package.json [hunk 2]
```diff
@@ -53,9 +53,9 @@
     }
   },
   "scripts": {
-    "test": "./node_modules/.bin/mocha --recursive test",
+    "test": "NODE_ENV=test ./node_modules/.bin/mocha --recursive test",
```
Comment (kethinov) [inline package.json]: does this work in windows?

**Pull request merged**